src/index.js

```diff
@@ -6,16 +6,21 @@ import { renderPreview, renderLoading, renderError, renderDisambiguation, render
 import { getWikipediaAttrFromUrl, isTouch, getDir, isOnline } from './utils'
 
 const invokeCallback = ( events, name, params ) => {
-	const callback = events && events[ name ]
-	if ( callback instanceof Function ) {
-		try {
-			callback.apply( null, params )
-		} catch ( e ) {
-			// eslint-disable-next-line no-console
-			console.log( 'Error invoking Wikipedia Preview custom callback', e )
+		const callback = events && events[ name ]
+		if ( callback instanceof Function ) {
+			try {
+				callback.apply( null, params )
+			} catch ( e ) {
+				// eslint-disable-next-line no-console
+				console.log( 'Error invoking Wikipedia Preview custom callback', e )
+			}
 		}
+	},
+
+	version = () => {
+	/* eslint-disable-next-line no-undef, no-console */
+		console.log( `Wikipedia Preview - ${APP_VERSION} (${CIRCLE_SHA1})` )
```

Comment:
Would it be useful to add the version as 'v1.1.1' or 'version 1.1.1'?

Reply:
I prefer the longer form (version 1.1.1)

Reply:
questions 

is it better to have this in the util?
when should the version number printed? init() or script loaded?

Reply:
I think it should print on script load just to know which version is included. The debug mode will provide info about init invocations.

Reply:
Alright, will invoke `version` outside of `init` within index.js. And I'll just make it a util as well, why not